config/routes.rb

```diff
@@ -20,12 +20,16 @@
 
   get "/trainees/not-supported-route", to: "trainees/not_supported_routes#index"
 
+  get "/sign-in" => "sign_in#index"
+  get "/signout" => "sign_out#index"
```

Comment:
Sorry... I meant this and the bit below should be `sign-out` to be consistent with `sign-in`

Reply:
fixed, thanks!